roles/bootstrap-os/tasks/bootstrap-oraclelinux.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Download Oracle Linux public yum repo
+  get_url:
+    url: "{{ repo_url }}"
+    dest: "{{ repo_dest }}"
+  vars:
```

Comment:
Why this construct with setting a variable per task?

Reply:
Yeah, makes sense to simplify it as there is only one repo.